Roblox Developers Rules.json

```diff
@@ -2,91 +2,53 @@
-              {
-                "color": 15687000,
-                "fields": [
-                  {
-                    "name": "Community safety comes first",
-                    "value": "We will prioritize the safety of our members first. If you threaten our member's livelihood, financial state, or security, you will be removed from the server."
-                  },
-                  {
-                    "name": "Always listen to moderators",
-                    "value": "These rules are non-exhaustive and cannot account for every possible scenario. If a moderator asks you to stop doing something, listen to them.\n\nAdministrators are still moderators and must also be listened to."
-                  },
-                  {
-                    "name": "Rules are subject to change",
-                    "value": "The rules above may change at any time and without prior notice. You are responsible for ensuring you are always complying with them. Ignorance is not an excuse.\n\nWe may post an announcement when the rules have been updated."
-                  },
-                  {
-                    "name": "Failure to comply may result in more severe punishments",
-                    "value": "Generally, we will warn you verbally to stop first. If you continue, you may be:\n\n- Given a more prominent warning that moderators can review in the future.\n- Muted so you are unable to talk in the server for an amount of time.\n- Roles may be taken away.\n- You may be kicked.\n- You may be banned.\n- Reported directly to authorities for further investigation in extreme circumstances or if required by Discord."
-                  },
-                  {
-                    "name": "How to appeal a ban",
-                    "value": "We provide a Discord server to appeal bans. If you have been banned, you can appeal at: https://discord.gg/k9jVfFD\n\nYou cannot appeal on someone else's behalf, this is a voluntary service and we will not guarantee any response times or actions."
-                  }
-                ],
-                "footer": {
-                  "text": "Roblox Developers - Moderation Team",
-                  "icon_url": "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/ROBLOX_Studio_icon.png/300px-ROBLOX_Studio_icon.png"
-                },
-                "timestamp": "2022-03-05T03:39:00.000Z"
-              }
-            ],
-            "username": "Rules",
-            "avatar_url": "https://cdn.discordapp.com/attachments/639845182686232581/849793385501294622/RBXDevLogo.png"
+          "description": "Please use __common sense__, this is **NOT** and exhaustive list of rules, and they are subject to change at any time.\nStaff reserve the right to punish for reasons not explicity listed here, if you would like to appeal a punishment you may open a ticket [here](https://discord.com/channels/489424959270158356/639890173869555761).\n\n__If you have any questions, do not hesitate to DM a staff member.__\nㅤ",
```

Comment:
Well I can change it to specify if they have been falsely punished, otherwise the only punishment that can really be *revoked* are bans which is taken care of in a separate server.